packages/wrangler/src/inspect.ts

```diff
@@ -326,68 +383,15 @@ export default function useInspector(props: InspectorProps) {
 					if (evt.method === "Runtime.exceptionThrown") {
 						const params = evt.params as Protocol.Runtime.ExceptionThrownEvent;
 
-						const sourceMapConsumer = await getSourceMapConsumer();
-						if (sourceMapConsumer !== undefined) {
+						const consumer = await getSourceMapConsumer();
+						if (consumer !== undefined) {
```

Comment:
Is there a reason for the variable name change, it went from very specific to generic?